lib/dry/schema/extensions/json_schema.rb

```diff
@@ -17,7 +17,7 @@ module SchemaMethods
         #
         # @api public
         def json_schema(loose: false)
-          compiler = SchemaCompiler.new(root: true, loose: loose)
+          compiler = SchemaCompiler.new(root: true, loose: loose, types: types)
```

Comment:
Need to pipe `types` down as those are where most of the meta / description info lives.